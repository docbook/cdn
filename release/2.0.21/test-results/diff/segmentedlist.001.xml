<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">segmentedlist</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">content</deltaxml:attributeValue></dxa:class></deltaxml:attributes>
            <div deltaxml:deltaV2="A" class="title">
               <strong>State Capitals</strong>
            </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
            </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">seglistitem</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">segmentedlist</deltaxml:attributeValue></dxa:class></deltaxml:attributes>
               <div deltaxml:deltaV2="B" class="title">
                  <strong>State Capitals</strong>
               </div><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
               </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">seg</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">seglistitem</deltaxml:attributeValue></dxa:class></deltaxml:attributes>
                  <strong deltaxml:deltaV2="A">
                     <span class="segtitle">State: </span>
                  </strong><div deltaxml:deltaV2="B" class="seg">
                     <strong>
                        <span class="segtitle">State: </span>
                     </strong>Alabama</div><deltaxml:textGroup deltaxml:deltaV2="A!=B"><deltaxml:text deltaxml:deltaV2="A">Alabama</deltaxml:text><deltaxml:text deltaxml:deltaV2="B">
                  </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" class="seg">
                     <strong>
                        <span class="segtitle">Capital: </span>
                     </strong>Montgomery</div><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
               </deltaxml:text></deltaxml:textGroup></div>
               <div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">seg</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">seglistitem</deltaxml:attributeValue></dxa:class></deltaxml:attributes>
                  <strong deltaxml:deltaV2="A">
                     <span class="segtitle">Capital: </span>
                  </strong><div deltaxml:deltaV2="B" class="seg">
                     <strong>
                        <span class="segtitle">State: </span>
                     </strong>Alaska</div><deltaxml:textGroup deltaxml:deltaV2="A!=B"><deltaxml:text deltaxml:deltaV2="A">Montgomery</deltaxml:text><deltaxml:text deltaxml:deltaV2="B">
                  </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" class="seg">
                     <strong>
                        <span class="segtitle">Capital: </span>
                     </strong>Anchorage</div><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
               </deltaxml:text></deltaxml:textGroup></div><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
               </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" class="seglistitem">
                  <div class="seg">
                     <strong>
                        <span class="segtitle">State: </span>
                     </strong>Arkansas</div>
                  <div class="seg">
                     <strong>
                        <span class="segtitle">Capital: </span>
                     </strong>Little Rock</div>
               </div>
            </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
            </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A" class="seglistitem">
               <div class="seg">
                  <strong>
                     <span class="segtitle">State: </span>
                  </strong>Alaska</div>
               <div class="seg">
                  <strong>
                     <span class="segtitle">Capital: </span>
                  </strong>Anchorage</div>
            </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
            </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A" class="seglistitem">
               <div class="seg">
                  <strong>
                     <span class="segtitle">State: </span>
                  </strong>Arkansas</div>
               <div class="seg">
                  <strong>
                     <span class="segtitle">Capital: </span>
                  </strong>Little Rock</div>
            </div>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>