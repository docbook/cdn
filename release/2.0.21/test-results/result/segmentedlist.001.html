<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for segmentedlist.001.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: segmentedlist.001.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">14</span> differences)</th><th>Actual (<span class="adiff">16</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">segmentedlist</span>"</span>&gt;
            <span class="tag diff del">&lt;div <span class="att">class="title"</span>&gt;
               <span class="tag">&lt;strong&gt;State Capitals&lt;/strong&gt;</span>
            &lt;/div&gt;</span><span class="diff txtchg">
            </span><span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">seglistitem</span>"</span>&gt;
               <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">seg</span>"</span>&gt;
                  <span class="tag diff del">&lt;strong&gt;
                     <span class="tag">&lt;span <span class="att">class="segtitle"</span>&gt;State: &lt;/span&gt;</span>
                  &lt;/strong&gt;</span><span class="diff txtchg">Alabama</span>&lt;/div&gt;</span>
               <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">seg</span>"</span>&gt;
                  <span class="tag diff del">&lt;strong&gt;
                     <span class="tag">&lt;span <span class="att">class="segtitle"</span>&gt;Capital: &lt;/span&gt;</span>
                  &lt;/strong&gt;</span><span class="diff txtchg">Montgomery</span>&lt;/div&gt;</span>
            &lt;/div&gt;</span><span class="diff txtchg">
            </span><span class="tag diff del">&lt;div <span class="att">class="seglistitem"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="seg"</span>&gt;
                  <span class="tag">&lt;strong&gt;
                     <span class="tag">&lt;span <span class="att">class="segtitle"</span>&gt;State: &lt;/span&gt;</span>
                  &lt;/strong&gt;</span>Alaska&lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="seg"</span>&gt;
                  <span class="tag">&lt;strong&gt;
                     <span class="tag">&lt;span <span class="att">class="segtitle"</span>&gt;Capital: &lt;/span&gt;</span>
                  &lt;/strong&gt;</span>Anchorage&lt;/div&gt;</span>
            &lt;/div&gt;</span><span class="diff txtchg">
            </span><span class="tag diff del">&lt;div <span class="att">class="seglistitem"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="seg"</span>&gt;
                  <span class="tag">&lt;strong&gt;
                     <span class="tag">&lt;span <span class="att">class="segtitle"</span>&gt;State: &lt;/span&gt;</span>
                  &lt;/strong&gt;</span>Arkansas&lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="seg"</span>&gt;
                  <span class="tag">&lt;strong&gt;
                     <span class="tag">&lt;span <span class="att">class="segtitle"</span>&gt;Capital: &lt;/span&gt;</span>
                  &lt;/strong&gt;</span>Little Rock&lt;/div&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">content</span>"</span>&gt;
            <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">segmentedlist</span>"</span>&gt;
               <span class="tag diff add">&lt;div <span class="att">class="title"</span>&gt;
                  <span class="tag">&lt;strong&gt;State Capitals&lt;/strong&gt;</span>
               &lt;/div&gt;</span><span class="diff txtchg">
               </span><span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">seglistitem</span>"</span>&gt;
                  <span class="tag diff add">&lt;div <span class="att">class="seg"</span>&gt;
                     <span class="tag">&lt;strong&gt;
                        <span class="tag">&lt;span <span class="att">class="segtitle"</span>&gt;State: &lt;/span&gt;</span>
                     &lt;/strong&gt;</span>Alabama&lt;/div&gt;</span><span class="diff txtchg">
                  </span><span class="tag diff add">&lt;div <span class="att">class="seg"</span>&gt;
                     <span class="tag">&lt;strong&gt;
                        <span class="tag">&lt;span <span class="att">class="segtitle"</span>&gt;Capital: &lt;/span&gt;</span>
                     &lt;/strong&gt;</span>Montgomery&lt;/div&gt;</span><span class="diff txtchg">
               </span>&lt;/div&gt;</span>
               <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">seglistitem</span>"</span>&gt;
                  <span class="tag diff add">&lt;div <span class="att">class="seg"</span>&gt;
                     <span class="tag">&lt;strong&gt;
                        <span class="tag">&lt;span <span class="att">class="segtitle"</span>&gt;State: &lt;/span&gt;</span>
                     &lt;/strong&gt;</span>Alaska&lt;/div&gt;</span><span class="diff txtchg">
                  </span><span class="tag diff add">&lt;div <span class="att">class="seg"</span>&gt;
                     <span class="tag">&lt;strong&gt;
                        <span class="tag">&lt;span <span class="att">class="segtitle"</span>&gt;Capital: &lt;/span&gt;</span>
                     &lt;/strong&gt;</span>Anchorage&lt;/div&gt;</span><span class="diff txtchg">
               </span>&lt;/div&gt;</span><span class="diff txtchg">
               </span><span class="tag diff add">&lt;div <span class="att">class="seglistitem"</span>&gt;
                  <span class="tag">&lt;div <span class="att">class="seg"</span>&gt;
                     <span class="tag">&lt;strong&gt;
                        <span class="tag">&lt;span <span class="att">class="segtitle"</span>&gt;State: &lt;/span&gt;</span>
                     &lt;/strong&gt;</span>Arkansas&lt;/div&gt;</span>
                  <span class="tag">&lt;div <span class="att">class="seg"</span>&gt;
                     <span class="tag">&lt;strong&gt;
                        <span class="tag">&lt;span <span class="att">class="segtitle"</span>&gt;Capital: &lt;/span&gt;</span>
                     &lt;/strong&gt;</span>Little Rock&lt;/div&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="segmentedlist"><div class="title"><strong>State Capitals</strong></div><div class="seglistitem"><div class="seg"><strong><span class="segtitle">State: </span></strong>Alabama</div><div class="seg"><strong><span class="segtitle">Capital: </span></strong>Montgomery</div></div><div class="seglistitem"><div class="seg"><strong><span class="segtitle">State: </span></strong>Alaska</div><div class="seg"><strong><span class="segtitle">Capital: </span></strong>Anchorage</div></div><div class="seglistitem"><div class="seg"><strong><span class="segtitle">State: </span></strong>Arkansas</div><div class="seg"><strong><span class="segtitle">Capital: </span></strong>Little Rock</div></div></div></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="content"><div class="segmentedlist"><div class="title"><strong>State Capitals</strong></div><div class="seglistitem"><div class="seg"><strong><span class="segtitle">State: </span></strong>Alabama</div><div class="seg"><strong><span class="segtitle">Capital: </span></strong>Montgomery</div></div><div class="seglistitem"><div class="seg"><strong><span class="segtitle">State: </span></strong>Alaska</div><div class="seg"><strong><span class="segtitle">Capital: </span></strong>Anchorage</div></div><div class="seglistitem"><div class="seg"><strong><span class="segtitle">State: </span></strong>Arkansas</div><div class="seg"><strong><span class="segtitle">Capital: </span></strong>Little Rock</div></div></div></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/segmentedlist.001.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>