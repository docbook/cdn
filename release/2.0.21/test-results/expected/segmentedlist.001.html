<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="segmentedlist"><div class="title"><strong>State Capitals</strong></div><div class="seglistitem"><div class="seg"><strong><span class="segtitle">State: </span></strong>Alabama</div><div class="seg"><strong><span class="segtitle">Capital: </span></strong>Montgomery</div></div><div class="seglistitem"><div class="seg"><strong><span class="segtitle">State: </span></strong>Alaska</div><div class="seg"><strong><span class="segtitle">Capital: </span></strong>Anchorage</div></div><div class="seglistitem"><div class="seg"><strong><span class="segtitle">State: </span></strong>Arkansas</div><div class="seg"><strong><span class="segtitle">Capital: </span></strong>Little Rock</div></div></div></article><script src="../../resources/js/prism.js"></script></body></html>